Nodejs/Product/Nodejs/Intellisense/IntellisenseController.cs

```diff
@@ -725,8 +732,12 @@ out title
                 || (ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9');
         }
 
+        private static bool IsIdentifierFirstChar(char ch) {
+            return ch == '_' || (ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z');
+        }

```

Comment:
Do such helper methods not already exist elsewhere in the parser code?

Also, interesting to see this is all ASCII only (not just your code changes, but surrounding code). Does NTVS not handle Unicode chars? (ES5 allows identifierNames as per the Unicode 3 standard - see http://es5.github.io/#x7.6 ).